### PR TITLE
feature: compatibility with angular 17

### PR DESCRIPTION
**Pull request by henriqueformiga**
Hi!

In this PR I'm adding support for Angular 17.

Thank you for this lib :)

### CODE REVIEW

**Review by aturoczy — APPROVED** [review]
+1

**Comment by WynieCronje** [comment]
When can we expect this release?

**Comment by EYALIN** [comment]
can we merge it?

**Comment by grozandrei** [comment]
can we?

**Comment by richard-elastique** [comment]
Hopefully this can be approved soon. :)

**Comment by w3sourcecode** [comment]
When can we expect this release?

File: projects/ngx-barcode6/package.json [hunk 1]
```diff
@@ -21,8 +22,8 @@
     "url": "https://github.com/efgiese/ngx-barcode6/issues"
   },
   "peerDependencies": {
-    "@angular/common": ">=14.0.0 <17.0.0",
-    "@angular/core": ">=14.0.0 <17.0.0",
+    "@angular/common": ">=14.0.0 <17.0.4",
```
Comment (dixy) [inline projects/ngx-barcode6/package.json:25]: It should be:
```suggestion
    "@angular/common": ">=14.0.0 <18.0.0",
```
Also we may be able to just have a minimum requirement:
```suggestion
    "@angular/common": ">=14.0.0",
```

Comment (henriqueformiga) [reply projects/ngx-barcode6/package.json]: @dixy version constraint removed, thanks!

**Comment by w3sourcecode** [comment]
Any news?

**Comment by elainesq** [comment]
When can we expect this PR to be merged?

**Comment by efgiese** [comment]
New version is out...

**Pull request closed without merging**